xml/System.AddIn.Hosting/AddInSecurityLevel.xml

```diff
@@ -80,16 +80,12 @@
       </ReturnValue>
       <MemberValue>3</MemberValue>
       <Docs>
-        <summary>Use the host security level to give the add-in the same permissions currently granted to the hosting code.</summary>
+        <summary>Use the host security level to give the add-in the same permissions currently granted to the hosting code. This setting has two purposes:<br/>- It represents the maximum permission set a host can grant to its add-ins.<br/>- It allows a ClickOnce or an Extensible Application Markup Language (XAML) Browser Application (XBAP) host to propagate its site of origin to its add-ins, which enables them to contact the server.</summary>
```

Comment:
same here re. para vs. br. This should also probably have a bulleted list using tags as well.